db/seeds/utils.js

```diff
@@ -20,3 +23,13 @@ exports.formatComments = (comments, idLookup) => {
     };
   });
 };
+
+
+exports.checkExists = (table, column, value) => {
+  const queryString = format(`SELECT * FROM %I WHERE %I = $1;`, table, column);
+  return db.query(queryString, [value]).then(({ rows }) => {
+    if (!rows.length) {
+      return Promise.reject({ status: 404, msg: "ID does not exist" });
+    }
+  });
+};
```

Comment:
This is a really reusable checkExists function, allowing you to pass in which table and property you want to use